### ISSUE TITLE
Bug Report: <short description>

### ISSUE THREAD

**Issue by jmorti** [stcr/subscribe-to-comments-reloaded#172]
The most recent update completely broke my comments notifications. The management pages dont even work correctly! The site is barnfinds.com. Should I just reinstall the old version until you get it fixed? Thanks.

**Comment by Reedyseth** [comment]
Hello @jmorti , please see this #159 There is a small change in the function invocation.

**Comment by Reedyseth** [comment]
Also you can take a look at this tutorial:

https://github.com/stcr/subscribe-to-comments-reloaded/wiki/KB#Real-Page

**Comment by jmorti** [comment]
What template file am I supposed to edit? Everything was working fine until you updated the code.

**Comment by Reedyseth** [comment]
Do you have the `Virtual Management Page` enable ?

**Comment by Reedyseth** [comment]
Also do you have access to the database ?

**Comment by Reedyseth** [comment]
I just subscribe to one post

![image](https://cloud.githubusercontent.com/assets/1228711/12311405/1027f806-ba14-11e5-87b4-39e9940a3eb0.png)

**Comment by Reedyseth** [comment]
I also notice that you have disable the `HTML emails`, please enable that in the options.

**Comment by Reedyseth** [comment]
In did there is an issue when you want to comment without subscribing, I would like to debug your site, let me know a time where is best to enable the debug flag of your wordpress config.

**Comment by Reedyseth** [comment]
I spot the issue and while release the fix asap. See #173

**Comment by jmorti** [comment]
I'm getting a blank page when trying to subscribe when I'm logged in. Also the manage page doesn't show any subscriptions. None of the existing subscriptions seem to be going out either. Please let me know what I need to do to fix this.

**Comment by Reedyseth** [comment]
Yes, there is a bug calling some code inside the plugin and I am working on
it !

On Thu, Jan 14, 2016 at 10:24 AM, jmorti <notifications@github.com> wrote:

> I'm getting a blank page when trying to subscribe when I'm logged in. Also
> the manage page doesn't show any subscriptions. None of the existing
> subscriptions seem to be going out either. Please let me know what I need
> to do to fix this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stcr/subscribe-to-comments-reloaded/issues/172#issuecomment-171713186>
> .
>



-- 

*Israel Barragán Castillo*Software Developer Engineer
Datasoft Engineering
Cel. +52(614) 345-5651
Web: www.datasoftengineering.com
Email: reedyseth@gmail.com

**Comment by Reedyseth** [comment]
Bug fixed, see #172, can you confirm that it is working after updating to the latest version?
